cbor/map.go

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+//go:build !tinygo
+
+package cbor
+
+import "reflect"
+
+func mapClear(rv reflect.Value) { rv.Clear() }
```

Comment:
I'm going to try to get `(reflect.Value).Clear` upstreamed to tinygo.

Reply:
This is turning into a very large free-time project. There is much of the reflect package not implemented that go-fdo needs.

Reply:
https://github.com/tinygo-org/tinygo/pull/4543